packages/suite/src/utils/wallet/reviewTransactionUtils.ts

```diff
@@ -175,11 +188,13 @@ const constructNewFlow = ({
     decreaseOutputId,
     account,
     precomposedForm,
-}: Omit<ConstructOutputsParams, 'device'>) => {
+    device,
+}: ConstructOutputsParams) => {
     const outputs: ReviewOutput[] = [];
 
     const isCardano = isCardanoTx(account, precomposedTx);
     const { networkType } = account;
+    const isWithUpdatedEthereumSendFlow = getIsUpdatedEthereumSendFlow(device, networkType);
```

Comment:
```suggestion
    const isUpdatedEthereumSendFlow = getIsUpdatedEthereumSendFlow(device, networkType);
```

The previous version is called `isUpdatedSendFlow` is our codebase